tests/unit/ingestion/models/ingestionManager.spec.ts

```diff
@@ -75,4 +86,33 @@ describe('IngestionManager', () => {
       await expect(ingestionManager.validateSources(inputFiles)).rejects.toThrow('Unexpected error');
     });
   });
+
+  describe('getFilesGdalInfoData', () => {
+    it('should return gdal info data when files exist and are valid', async () => {
+      const inputFiles = fakeIngestionSources.validSources.validInputFiles;
+      const mockGdalInfoData = [gdalInfoCases.validGdalInfo];
+
+      sourceValidator.validateFilesExist.mockImplementation(async () => Promise.resolve());
+      gdalInfoManagerMock.getFilesGdalInfoData.mockResolvedValue(mockGdalInfoData);
+
+      const result = await ingestionManager.getFilesGdalInfoData(inputFiles);
+
+      expect(result).toEqual(mockGdalInfoData);
+    });
+
+    it('should throw an error when validateFilesExist throws FileNotFoundError', async () => {
+      const inputFiles = fakeIngestionSources.invalidSources.filesNotExist;
+      sourceValidator.validateFilesExist.mockImplementation(async () => Promise.reject(new FileNotFoundError(inputFiles.fileNames[0])));
+
+      await expect(ingestionManager.getFilesGdalInfoData(inputFiles)).rejects.toThrow(FileNotFoundError);
+    });
+
+    it('should throw an error when getFilesGdalInfoData throws GdalInfoError', async () => {
+      const inputFiles = fakeIngestionSources.invalidSources.unsupportedCrs;
```

Comment:
unsupportedCrs should return valid gdal infor with the 'invalid' crs

Reply:
agree, changed